src/statmodels.jl

```diff
@@ -446,21 +446,36 @@ struct ConvergenceException{T<:Real} <: Exception
     iters::Int
     lastchange::T
     tol::T
-    function ConvergenceException{T}(iters, lastchange::T, tol::T) where T<:Real
+    msg::String
+    function ConvergenceException{T}(
```

Comment:
Please use the same style as elsewhere, with arguments on the same line as parens.

Reply:
If everything is on the same line, it's 92 characters long. 80 characters is the usual line length for source code. Where else would you like me to put a line break?

Reply:
92 chars is the limit we use (as in Julia). Anyway you can perfectly break the line if needed even with that convention.

Reply:
I put it all on the same line.